dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/MaintenanceController.java

```diff
@@ -167,6 +167,15 @@ public void deleteSoftDeletedProgramStageInstances()
         maintenanceService.deleteSoftDeletedProgramStageInstances();
     }
 
+    @RequestMapping( value = "/softDeletedRelationshipRemoval", method = { RequestMethod.PUT,
+        RequestMethod.POST } )
+    @PreAuthorize( "hasRole('ALL') or hasRole('F_PERFORM_MAINTENANCE')" )
+    @ResponseStatus( HttpStatus.NO_CONTENT )
```

Comment:
A bit odd that we don't return a bit of info on the operation like how many deletions took place. Since we don't do this for the other endpoints its consistent not to return anything here 👌🏻